src/Log/AccessLogDataMap.php

```diff
@@ -94,7 +95,12 @@ public static function createWithStaticResource(
      */
     public function getClientIp() : string
     {
-        return $this->getLocalIp();
+        $headers = ['x-forwarded-for', 'client-ip', 'x-real-ip'];
+        $headerValue = array_reduce($headers, function (?string $default, string $header) {
```

Comment:
can use `static function`, eg:

```suggestion
        $headerValue = array_reduce($headers, static function (?string $default, string $header) {
```

Reply:
Actually you cannot in that case. `$this` is used inside.

Reply:
@michalbundyra oh yeah, just realize that.